worlds/celeste64/__init__.py

```diff
@@ -0,0 +1,95 @@
+from typing import List
+
+from BaseClasses import ItemClassification, Region, Tutorial
+from worlds.AutoWorld import WebWorld, World
+from .Items import Celeste64Item, item_data_table, item_table
+from .Locations import Celeste64Location, location_data_table, location_table
+from .Names import ItemName
+from .Options import Celeste64Options
+
+
+class Celeste64WebWorld(WebWorld):
+    theme = "ice"
+
+    setup_en = Tutorial(
+        tutorial_name="Start Guide",
+        description="A guide to playing Celeste 64 in Archipelago.",
+        language="English",
+        file_name="guide_en.md",
+        link="guide/en",
+        authors=["PoryGone"]
+    )
+
+    tutorials = [setup_en]
+
+
+class Celeste64World(World):
+    """Relive the magic of Celeste Mountain alongside Madeline in this small, heartfelt 3D platformer.
+    Created in a week(ish) by the Celeste team to celebrate the game’s sixth anniversary 🍓✨"""
+
+    game = "Celeste 64"
+    data_version = 0
+    web = Celeste64WebWorld()
+    options_dataclass = Celeste64Options
+    options: Celeste64Options
+    location_name_to_id = location_table
+    item_name_to_id = item_table
+
+
+    def create_item(self, name: str) -> Celeste64Item:
+        # Only make required amount of strawberries be Progression
+        if getattr(self, "options", None) and name == ItemName.strawberry:
+            classification: ItemClassification = ItemClassification.filler
+            self.prog_strawberries = getattr(self, "prog_strawberries", 0)
+            if self.prog_strawberries < self.options.strawberries_required.value:
+                classification = ItemClassification.progression
+                self.prog_strawberries += 1
+
+            return Celeste64Item(name, classification, item_data_table[name].code, self.player)
+        else:
+            return Celeste64Item(name, item_data_table[name].type, item_data_table[name].code, self.player)
+
+    def create_items(self) -> None:
+        item_pool: List[Celeste64Item] = []
+
+        for name, item in item_data_table.items():
+            item_pool.append(self.create_item(name))
```

Comment:
think i'd personally prefer just adding to the multiworld itempool directly rather than using an intermediary but nbd.
```suggestion
        item_pool += [self.create_item(name) for name in item_data_table]
```